src/Rector/ValueObject/StaticArgumentRenameConfiguration.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace DrupalRector\Rector\ValueObject;
+
+class StaticArgumentRenameConfiguration {
+
+    protected string $methodName;
+
+    protected string $fullyQualifiedClassName;
+
+    protected string $newArgument;
+
+    protected string $oldArgument;
+
+    public function __construct(string $oldArgument, string $newArgument, string $fullyQualifiedClassName = 'Drupal', string $methodName = 'service') {
```

Comment:
Should this default to `\Drupal::service`? I guess maybe that's what "static" refers to? Should it be `StaticServiceArgument...` to be more clear?

Reply:
When built there was a generic base class for the drupal service and one above that.  We could just make this about Drupal service and remove the  premature abstraction.

Reply:
Renames class and arguments to something more sane.